src/sentry/search/events/datasets/spans_indexed.py

```diff
@@ -194,6 +194,40 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     result_type_fn=self.reflective_result_type(),
                     redundant_grouping=True,
                 ),
+                SnQLFunction(
+                    "example",
+                    snql_aggregate=lambda args, alias: Function(
+                        "argMin",
+                        [
+                            Function("tuple", [Column("group"), Column("span_id")]),
+                            Function("cityHash64", [Column("span_id")]),
+                        ],
+                        alias,
+                    ),
+                ),
```

Comment:
Could you do me a favour and add a comment explaining the other functions are deprecated for these functions now?